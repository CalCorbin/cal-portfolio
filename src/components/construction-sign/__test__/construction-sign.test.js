import React from 'react'
import { cleanup, render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import renderer from 'react-test-renderer'
import Construction from '../index'

afterEach(cleanup)

it('should render the construction sign correctly', () => {
  const { getByTestId } = render(<Construction />)
  expect(getByTestId('construction-sign')).toHaveClass('__construction-sign')
})

it('should match the construction sign snapshot', () => {
  const tree = renderer.create(<Construction />).toJSON()
  expect(tree).toMatchSnapshot()
})
