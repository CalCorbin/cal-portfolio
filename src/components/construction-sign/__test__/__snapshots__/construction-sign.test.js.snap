// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the construction sign snapshot 1`] = `
<div>
  <div
    className="__construction-sign"
    data-testid="construction-sign"
  >
    <p>
      Currently under construction
    </p>
    <a
      className="link"
      href="https://www.linkedin.com/in/calcorbin/"
      rel="noopener noreferrer"
      target="_blank"
    >
      Reach me here for now
    </a>
  </div>
</div>
`;
